news/2019-10-01-new-featured-artist-updates.md

```diff
@@ -0,0 +1,247 @@
+---
+layout: post
+title: "Featured Artist Track Updates: October 2019"
+date: 2019-10-01 10:00:00 +0000
+---
+
+We promised a big update in the last Featured Artist reveal, and boy did we mean it. **197** new tracks from **12** of our absurdly talented Featured Artists are now available. Check it all out!
+
+There's so much to go over, so we'll dip right into it - just remember that **clicking the artist banners will take you directly to their Featured Artist listing**, and that the newer tracks listed in this post are likely to be towards the bottom of their respective page.
+
+Got that? Great. ONWARDS!
+
+## antiPLUR / Internet Death Machine
+
+[![](https://assets.ppy.sh/artists/24/header.jpg)](https://osu.ppy.sh/beatmaps/artists/24)
+
+New tracks from **antiPLUR**:
```

Comment:
```suggestion
New tracks as **antiPLUR**:
```
to distinguish that you're talking about an artist name here, not an album title ?
would be same for Internet Death Machine if u want to change thsi